### ISSUE TITLE
[Core] 🐛 Use a href HTML tag in a slide give an error

### ISSUE THREAD

**Issue by philippart-s** [gouz/slidesk#87]
👋,

If I add an `<a href>` HTML tag in a SDF file I have an error.

The slide source code:
```html
## Merciii !!! .[love]

<div><a href="https://unsplash.com/@bel2000a">Belinda Fewings</a></div>

<img src="assets/images/love.jpg" width="70%" alt="" class=""/>
```

The error:
```bash
error: Parse Error: <a href="<a href="https://unsplash.com/@bel2000a">Belinda" target="_blank" rel="noopener">https://unsplash.com/@bel2000a">Belinda</a> Fewings</a></div>
```

It appears that the core engine try to _translate_ the link `https://unsplash.com/@bel2000a` in a `<a href>` HTML tag inside my `<a href>` HTML tag 😉.

**Comment by gouz** [comment]
Fix with #89

**Comment by gouz** [comment]
So, upgrade SliDesk to 2.2.8
